pdu.go

```diff
@@ -26,18 +26,18 @@ type PDU interface {
 	HistoryVisibility() (HistoryVisibility, error)
 	Membership() (string, error)
 	PowerLevels() (*PowerLevelContent, error)
-	Version() RoomVersion
-	RoomID() string
-	Redacts() string
+	RoomVersion() RoomVersion
+	GetRoomID() string
+	GetRedacts() string
```

Comment:
Please keep the old naming and instead rename fields on `ProtoEvent`. The reason for this is that GMSL callers overwhelmingly use `PDU` and not `ProtoEvent`, so `PDU` should have more ergonomic names than `ProtoEvent` when they are conflicting.